# HG changeset patch
# User Torsten <ttl@justmail.de>
# Date 1451508552 -3600
#      Wed Dec 30 21:49:12 2015 +0100
# Node ID cb1eb7d9a89fffb53837c96c7e504db153081c56
# Parent  93748bcaec17bc7540d1c0f96c532ae71643545c
resolve some identifier ambiguities in the settings dialog

* settings-dialog.ui: rename some identifiers in order to resolve duplicates

diff --git a/libgui/src/settings-dialog.ui b/libgui/src/settings-dialog.ui
--- a/libgui/src/settings-dialog.ui
+++ b/libgui/src/settings-dialog.ui
@@ -1498,17 +1498,17 @@
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_debugger">
       <attribute name="title">
        <string>Debugger</string>
       </attribute>
-      <layout class="QVBoxLayout" name="verticalLayout_30">
+      <layout class="QVBoxLayout" name="vlayout_dbg_00">
        <item>
         <widget class="QScrollArea" name="scrollArea_8">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
@@ -1519,23 +1519,23 @@
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>658</width>
             <height>571</height>
            </rect>
           </property>
-          <layout class="QVBoxLayout" name="verticalLayout_28">
+          <layout class="QVBoxLayout" name="vlayout_dbg_01">
            <item>
-            <layout class="QVBoxLayout" name="verticalLayout_13">
+            <layout class="QVBoxLayout" name="vlayout_dbg_02">
              <item>
-              <layout class="QGridLayout" name="gridLayout_15">
+              <layout class="QGridLayout" name="glayout_dbg_01">
                <item row="4" column="0">
-                <widget class="QGroupBox" name="groupBox_6">
+                <widget class="QGroupBox" name="groupbox_dbg_01">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="title">
                   <string>Breakpoint Retention Upon Saving File</string>
@@ -1581,17 +1581,17 @@
                  </widget>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
-            <spacer name="verticalSpacer_7">
+            <spacer name="vspacer_dbg_01">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
